pages/advanced/smart-account-overview.mdx

```diff
@@ -3,8 +3,23 @@ import SafeCoreSDK from '../../assets/svg/ic-sdk.svg'
 import SafeSmartAccount from '../../assets/svg/ic-smart-account.svg'
 import CustomCard from '../../components/CustomCard'
 
-# Safe Smart Account
+## EOAs
```

Comment:
Having EOAs as the page title is a bit confusing for me. How about something like `How do Safe Smart Accounts work?` and the EOAs as H2?

Reply:
https://github.com/safe-global/safe-docs/pull/653/commits/43f9f5d563e5d25cb184f9eef1dd2f54ad10b99a